x11-libs/pango/pango-1.50.14.recipe

```diff
@@ -41,11 +41,13 @@ REQUIRES="
 	lib:libfontconfig$secondaryArchSuffix
 	lib:libfreetype$secondaryArchSuffix
 	lib:libfribidi$secondaryArchSuffix
-	lib:libgirepository_1.0$secondaryArchSuffix
+	lib:libgio_2.0$secondaryArchSuffix
+#	lib:libgirepository_1.0$secondaryArchSuffix
 	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
 	lib:libharfbuzz$secondaryArchSuffix
-	lib:libiconv$secondaryArchSuffix
-	lib:libintl$secondaryArchSuffix
+#	lib:libiconv$secondaryArchSuffix
+#	lib:libintl$secondaryArchSuffix
```

Comment:
Disabled ones don't show up as warning/error policy when completing the build, so they don't seem to be a hard requirement.